src/Analyzer.PowerShellRuleEngine/Analyzer.PowerShellRuleEngine.csproj

```diff
@@ -5,13 +5,13 @@
     <AssemblyName>Microsoft.Azure.Templates.Analyzer.PowerShellRuleEngine</AssemblyName>
     <RootNamespace>Microsoft.Azure.Templates.Analyzer.RuleEngines.PowerShellEngine</RootNamespace>
   </PropertyGroup>
-  
+
   <ItemGroup>
     <PackageReference Include="Microsoft.PowerShell.SDK" Version="7.2.4" />
     <PackageReference Include="Microsoft.PSRule.Rules.Azure" Version="1.22.0-B0011" />
```

Comment:
Wait, was PSRule updated or not? This is unchanged but I see new rules in the baseline file

Reply:
Oh no this must have gotten messed up during merge or something, I will need to retest and see if the issues I had were actually fixed or not.

Reply:
We are ok with the tests. Thanks for catching.